docs/reference/connector/apis/update-connector-configuration-api.asciidoc

```diff
@@ -19,7 +19,8 @@ Updates a connector's `configuration`, allowing for complete schema modification
 
 * To sync data using self-managed connectors, you need to deploy the {enterprise-search-ref}/build-connector.html[Elastic connector service] on your own infrastructure. This service runs automatically on Elastic Cloud for native connectors.
 * The `connector_id` parameter should reference an existing connector.
-* The configuration fields definition must be compatible with the specific connector type being used.
+* To update configuration `values`, the connector `configuration` schema must be first registered by a running instance of Elastic connector service.
```

Comment:
This could be clearer. I'd try to answer the question, _"what actions does a user have to do explicitly versus what is handled behind the scenes?"_. 

What does a user have to do before they can update the values? Do they just need to have the connector service running?

Reply:
> What does a user have to do before they can update the values? Do they just need to have the connector service running?

They need to have the configured service running:
- for native connectors the schema will be populated by the connector service automatically 
- for self-managed connectors, you need to populate `config.yml` and start the service, then schema will be populated automatically 

Do you think this-mini tutoriual deserves separate section on this page? I know users were confuser by this often

Reply:
Same as similar response elsewhere, I think we're covered by saying you need to have the connector service running, we mightn't need to go into every detail

Reply:
So I think we can delete this line